server/lobbyconnection.py

```diff
@@ -190,7 +188,7 @@ async def on_message_received(self, message):
             await self.abort("Error processing command")
 
     async def command_ping(self, msg):
-        await self.protocol.send_raw(PONG_MSG)
+        await self.send({"command": "pong"})
```

Comment:
How much of an impact does this have? I see that you are still caching `PING` but no longer `PONG` for every protocol (although both are cached inside `QDataStreamProtocol`)?.

Reply:
Yea it will create a temporary dict every time, which is pretty negligible (I benchmarked a similar thing a while ago). It could be refactored but I don’t think it’s worth it. Just a lot nicer to read this way